assets/css/style.css

```diff
@@ -639,3 +639,25 @@ header {
 .paragraph {
   font-size: 21px;
 }
+
+/* Styling the form */
+.form {
+  display: flex;
+  justify-content: space-between;
+}
+
+input {
```

Comment:
Be careful, this way you are targetting all the inputs from the page. It's better to just target the inputs inside the form, like this: `.form input { }`. I recommend you to change it.

Reply:
Got it! thanks.